modules/user_data/templates/start_graphdb.sh.tpl

```diff
@@ -16,126 +16,41 @@ instance_id=$( curl -Ss -H "X-aws-ec2-metadata-token: $imds_token" 169.254.169.2
-device_mapping_full="${device_name}"
-device_mapping_short="$(echo $device_mapping_full | cut -d'/' -f3)"
-
-graphdb_device=""
-
-# the device might not be available immediately, wait a while
-for i in $(seq 1 12); do
-  for volume in $(find /dev | grep -i 'nvme[0-21]n1$'); do
-    # extract the specified device from the vendor-specific data
-    # read https://github.com/oogali/ebs-automatic-nvme-mapping/blob/master/README.md, for more information
-    real_device=$(nvme id-ctrl --raw-binary $volume | cut -c3073-3104 | tr -s ' ' | sed 's/ $//g')
-    if [ "$device_mapping_full" = "$real_device" ] || [ "$device_mapping_short" = "$real_device" ]; then
-      graphdb_device="$volume"
-      break
-    fi
-  done
-
-  if [ -n "$graphdb_device" ]; then
-    break
-  fi
-  sleep 5
-done
-
-# create a file system if there isn't any
-if [ "$graphdb_device: data" = "$(file -s $graphdb_device)" ]; then
-  mkfs -t ext4 $graphdb_device
-fi
+# Defining variables by interpolating Terraform variables
 
-disk_mount_point="/var/opt/graphdb"
+region="${region}"
+name="${name}"
+device_name="${device_name}"
 
-# Check if the disk is already mounted
-if ! mount | grep -q "$graphdb_device"; then
-  echo "The disk at $graphdb_device is not mounted."
+backup_schedule="${backup_schedule}"
+backup_retention_count="${backup_retention_count}"
+backup_bucket_name="${backup_bucket_name}"
 
-  # Create the mount point if it doesn't exist
-  if [ ! -d "$disk_mount_point" ]; then
-    mkdir -p "$disk_mount_point"
-  fi
+ebs_volume_type="${ebs_volume_type}"
+ebs_volume_size="${ebs_volume_size}"
+ebs_volume_iops="${ebs_volume_iops}"
+ebs_volume_throughput="${ebs_volume_throughput}"
+ebs_kms_key_arn="${ebs_kms_key_arn}"
 
-  # Add an entry to the fstab file to automatically mount the disk
-  if ! grep -q "$graphdb_device" /etc/fstab; then
-    echo "$graphdb_device $disk_mount_point ext4 defaults 0 2" >> /etc/fstab
-  fi
+zone_dns_name="${zone_dns_name}"
+zone_id="${zone_id}"
 
-  # Mount the disk
-  mount "$disk_mount_point"
-  echo "The disk at $graphdb_device is now mounted at $disk_mount_point."
-else
-  echo "The disk at $graphdb_device is already mounted."
-fi
+jvm_max_memory="${jvm_max_memory}"
+resource_name_prefix="${resource_name_prefix}"
 
-# Ensure data folders exist
-mkdir -p $disk_mount_point/node $disk_mount_point/cluster-proxy
+GRAPHDB_CONNECTOR_PORT=""
 
-# this is needed because after the disc attachment folder owner is reverted
-chown -R graphdb:graphdb $disk_mount_point
+# Search for an available EBS volume to attach to the instance. If no volume is found - create new one, attach, format and mount the volume.
+#source /opt/helper-scripts/ebs_volume.sh
+./opt/helper-scripts/ebs_volume.sh --name ${nam e} --ebs_volume_type ${ebs_volume_type} --ebs_volume_throughput ${ebs_volume_throughput} --ebs_kms_key_arn ${ebs_kms_key_arn} --ebs_volume_size ${ebs_volume_size} --ebs_volume_iops ${ebs_volume_iops} --device_name ${device_name}
```

Comment:
Please make each property go on a new line e.g.,
```
./opt/helper-scripts/ebs_volume.sh --name ${name} \
--ebs_volume_type ${ebs_volume_type} \
....... \
....... \
.......
```
Would be easier to read.

Reply:
Done.